Packs/CrowdStrikeHost/Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -483,6 +483,74 @@ script:
         };
     }
 
+    function threatGraphSummary() {
+        var ids = argToList(args.ctg_id);
+        var res = doReq('GET', 'threatgraph/combined/control-graphs/summary/v1', {ids: ids});
+        var md = '## Falcon Host Threat Graph Summary\n';
+        var ec = {};
+        if (res.obj.resources && res.obj.resources.length > 0) {
+            ec= {"CrowdStrikeHost.ThreatGraphSummary": res.obj.resources};
+            for (var i=0; i<res.obj.resources.length; i++) {
+               md += objToList(res.obj.resources[i], res.obj.resources[i].value)
+
+            }
+        } else {
+            md += 'No result found';
+        }
+        return {
+            Type: entryTypes.note,
+            Contents: res.obj,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            EntryContext: ec
+        };
+    }
+
+    function threatGraphProcesses() {
+        var ids = argToList(args.id);
+        var res = doReq('GET', 'threatgraph/combined/processes/summary/v1', {ids: ids});
+        var md = '## Falcon Host Threat Graph Processess Summary\n';
+        var ec = {};
+        if (res.obj.resources && res.obj.resources.length > 0) {
+            ec= {"CrowdStrikeHostThreatGraphProcessess": res.obj.resources};
```

Comment:
no merging DT. maybe there is an ID we can use to merge? see other commands implementation.